mg/deps.go

```diff
@@ -151,10 +159,17 @@ func name(i interface{}) string {
 	return runtime.FuncForPC(reflect.ValueOf(i).Pointer()).Name()
 }
 
+func displayName(name string) string {
```

Comment:
I think it might be useful to print the package, actually.  Targets get printed without the package name, but by definition they have to be in the main package. Dependencies don't.    Maybe if the package is "main" then don't print the package, but otherwise, do?

Reply:
Good remark, I was mostly thinking of `main` indeed!